src_cpp/elfgames/go/common/go_state_ext.cc

```diff
@@ -82,47 +82,52 @@ std::string GoStateExt::dumpSgf(const std::string& filename) const {
 
 void GoStateExt::showFinishInfo(FinishReason reason) const {
   Stone player = _state.nextPlayer();
-  std::cout << _state.showBoard() << std::endl;
+  _logger->info("{}\n", _state.showBoard());
```

Comment:
Logger will automatically add a new line so `\n` is typically unnecessary. Same for all other lines below.